setup_board

```diff
@@ -155,6 +155,12 @@ fi
 if [ "${FLAGS_skip_toolchain_update}" -eq "${FLAGS_TRUE}" ]; then
   UPDATE_ARGS+=" --skip_toolchain_update"
 fi
+if [[ "${BOARD}" == "arm64-usr" ]]; then
+  [[ ! -d /etc/portage/package.use ]] && sudo mkdir /etc/portage/package.use
+  sudo_clobber "/etc/portage/package.use/grub" <<EOF
+sys-boot/grub grub_platforms_arm64
+EOF
+fi
```

Comment:
I think it would be OK (and simpler) to just build the arm64 modules unconditionally by adding arm64 to GRUB_PLATFORMS in coreos-overlay/profiles/coreos/base/make.defaults.

Would there be any problem in doing it that way?